api.md

```diff
@@ -0,0 +1,212 @@
+
+### Transactions
+
+- [`/transactions`](#get-apiv1transactions)
+- [`/transactions/{hash}`](#get-apiv1transactionshash)
+
+#### `GET` `/api/v1/transactions`
+
+- **Query**: [Pagination](#pagination-query-params)
+- **Response**: `Paginated<Transaction>`
+
+#### `GET` `/api/v1/transactions/{hash}`
+
+- **Params**:
+
+  | Param  |   Type   |             Description             |
+  | :----: | :------: | ----------------------------------- |
+  | `hash` | `string` | 32-byte hash hex of the transaction |
+
+- **Response**: `Transaction` or `404`
+
+### Accounts
+
+- [`/accounts`](#get-apiv1accounts)
+- [`/accounts/{id}`](#get-apiv1accountsid)
+
+#### `GET` `/api/v1/accounts`
+
+- **Query**: [Pagination](#pagination-query-params)
+- **Response**: `Paginated<Account>`
+
+#### `GET` `/api/v1/accounts/{id}`
+
+- **Params**:
+
+  | Param |   Type   |      Description      |
+  | :---: | :------: | --------------------- |
+  | `id`  | `string` | The id of the account |
+
+- **Response**: `Account` or `404`
+
+See also: [Id Transformation](#id-transformation)
+
+### Assets
+
+- [`/assets`](#get-apiv1assets)
+- [`/assets/{definition_id}/{account_id}`](#get-apiv1assetsdefinition_idaccount_id)
+
+#### `GET` `/api/v1/assets`
+
+- **Query**: [Pagination](#pagination-query-params)
+- **Response**: `Paginated<Asset>`
+
+#### `GET` `/api/v1/assets/{definition_id}/{account_id}`
+
+- **Params**:
+
+  |      Param      |   Type   |                Description                 |
+  | :-------------: | :------: | ------------------------------------------ |
+  | `definition_id` | `string` | The id of the asset definition             |
+  |  `account_id`   | `string` | The id of the account the asset belongs to |
+
+- **Response**: `Asset` or `404`
+
+See also: [Id Transformation](#id-transformation)
+
+### Asset Definitions
+
+- [`/asset-definitions`](#get-apiv1asset-definitions)
+- [`/asset-definitions/{id}`](#get-apiv1asset-definitionsid)
+
+#### `GET` `/api/v1/asset-definitions`
+
+- **Query**:[Pagination](#pagination-query-params)
+- **Response**: `Paginated<AssetDefinition>`
+
+#### `GET` `/api/v1/asset-definitions/{id}`
+
+- **Params**:
+
+  | Param |   Type   |          Description           |
+  | :---: | :------: | ------------------------------ |
+  | `id`  | `string` | The id of the asset definition |
+
+- **Response**: `AssetDefinitionWithAccounts` or 404
+
+See also: [Id Transformation](#id-transformation)
+
+### Domains
+
+- [`/domains`](#get-apiv1domains)
+- [`/domains/{id}`](#get-apiv1domainsid)
+
+#### `GET` `/api/v1/domains`
+
+- **Query**: [Pagination](#pagination-query-params)
+- **Response**: `Paginated<Domain>`
+
+#### `GET` `/api/v1/domains/{id}`
+
+- **Params**:
+
+  | Param |   Type   |     Description      |
+  | :---: | :------: | -------------------- |
+  | `id`  | `string` | The id of the domain |
+
+- **Response**: `Domain` or `404`
+
+### Peers
+
+- [`/peer/peers`](#get-apiv1peerpeers)
+- [`/peer/status`](#get-apiv1peerstatus)
```

Comment:
I'd name this section "Peer data" (the data of the concrete peer that the explorer is connected to) instead of "Peers"

Reply:
well it's "data" for all sections, not only this one, isn't it? this is only to indicate which entity the queries are related to. I think if we add "data" here, we'd need to do it to all other sections to be consistent

Reply:
I agree about "data", but I think the main point is "Peer" instead of "Peers".

Reply:
changed to "Peer", ok?

Reply:
Ok!